test/e2e/specs/editor/various/new-post-default-content.spec.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'new editor filtered state', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.activatePlugin(
+			'gutenberg-test-plugin-default-post-content'
+		);
+	} );
+
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test.afterAll( async ( { requestUtils } ) => {
+		await requestUtils.deactivatePlugin(
+			'gutenberg-test-plugin-default-post-content'
+		);
+	} );
+
+	test( 'should respect default content', async ( { editor, page } ) => {
+		const content = await editor.getEditedPostContent();
+
+		await page.click( 'role=button[name="Excerpt"i]' );
```

Comment:
```suggestion
		await editor.openDocumentSettingsSidebar();
		await page.click( 'role=button[name="Excerpt"i]' );
```

Let's make sure the "Document Sidebar" is open before we perform any actions there.